packages/client/src/hooks/use-agent-update.ts

```diff
@@ -23,6 +23,41 @@ export function useAgentUpdate(initialAgent: Agent) {
     updateSettings,
   } = usePartialUpdate(initialAgent);
 
+  // ==================== Template Import Function ====================
+  /**
+   * Imports a full agent template - overwrites all fields
+   *
+   * @param templateAgent The agent template to import
+   */
+  const importAgent = useCallback(
```

Comment:
Have we tested this hook to make sure no values in the character.json get missed if not we should add a quick test for this - should be pretty straightforeward

Reply:
Thanks